functionary/ui/templates/core/function_list.html

```diff
@@ -1,38 +1,5 @@
 {% extends "base.html" %}
 {% block content %}
-    <div>
-        {% for function in object_list %}
-            <div class="block">
-                <div class="card">
-                    <div class="card-content">
-                        <div class="media">
-                            <div class="media-left">
-                                <span class="icon is-large">
-                                    <i class="fa fa-2x fa-cube"></i>
-                                </span>
-                            </div>
-                            <div class="media-content">
-                                <a href="{% url 'ui:function-detail' function.id %}">
-                                    <p class="title is-4">
-                                        {% if function.display_name %}
-                                            {{ function.display_name }} ({{ function.name }})
-                                        {% else %}
-                                            {{ function.name }}
-                                        {% endif %}
-                                    </p>
-                                </a>
-                                <p class="subtitle is-6">
-                                    <a class="has-text-info"
-                                       href="{% url 'ui:package-detail' function.package.id %}">
-                                        <span class="icon"><i class="fa fa-cubes"></i></span>&nbsp;{{ function.package.name|lower }}
-                                    </a>
-                                </p>
-                                {% if function.summary %}<p class="subtitle is-6">{{ function.summary }}</p>{% endif %}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        {% endfor %}
-    </div>
+    {% include "partials/functions/list.html" %}
```

Comment:
What's the motivation behind the second layer here with the partial as opposed to just moving the contents of that partial here?  Is the thought that we might want to embed the full table in other places?  If so, where? If there isn't a compelling example use case I'd vote to remove the partials and just put the contents in each of these list templates directly.

Reply:
Since the tables now work the same as modals, you need to extend from them to fill out the table header and row blocks. With Django templates, you cannot extend from more than one file, so I either had to abandon the table partial, which would have been annoying to refactor in the future if we wanted to reformat the tables, or I could do what I did and just include a partial that extends from the table partial. Since we are planning on switching css libraries in the near future, I felt that the second option made more sense to make that refactor less painful

Reply:
I wonder if this is more of a custom template tag situation then...Carry on with how you have it organized for now and maybe we can play around with other ideas in the future.